webapp/src/components/Profile.tsx

```diff
@@ -27,95 +31,119 @@ const user = {
 };
 
 const Profile = () => {
+  let navigate = useNavigate();
```

Comment:
Why do we use the UseNavigation hook here?
What if use button component with href attribute instead:
`<Button
            component="a"
            sx={{ mr: 1, height: '40px' }}
            href={'/competencies'}
  >Competencies</Button>`

Reply:
I checked if the way you suggested works and it does. We will discuss which way is better with others on Friday.

Reply:
I changed useNavigate hook to buttons with href attribute